backend/src/routers/gameRouter.ts

```diff
@@ -346,9 +358,52 @@ gameRouter.post('/markPlayed', async (req, res) => {
 		res.status(200).json({ message: 'Game marked as played' });
 	} catch (e) {
 		if (e instanceof Error)
-			res.status(400).json({ message: e.message });
+			res.status(500).json({ message: e.message });
+		else
+			res.status(500).json({ message: 'Error marking game as played' });
+	}
+});
+
+/**
+ * @api {post} /api/v1/games/markNotPlayed Saves that a user has played a game
+ * @apiName markPlayed
+ * @apiGroup Games
+ * @apiDescription Marks the game as played for the user
+ *
+ * @apiBody {String} gameId Id of the game
+ * @apiBody {String} userId Id of the user
+ *
+ * @apiSuccess {String} message Message indicating success
+ *
+ * @apiSuccessExample Success-Response:
+ * HTTP/1.1 200 OK
+ *   {
+ *    "message": "Game marked as not played"
+ *   }
+ * @apiErrorExample {json} Unauthorized:
+ * HTTP/1.1 401 Unauthorized
+ *   {
+ *    "message": "Unauthorized"
+ *   }
+ *
+ * @apiErrorExample {json} Internal server Error:
+ * HTTP/1.1 500 Internal server error
+ *   {
+ *    "message": "Error marking game as not played"
+ *   }
+ *
+ * @apiUse ZodError
+ */
+gameRouter.post('/markNotPlayed', async (req, res) => {
```

Comment:
I know this is how we do `markAsPlayed` but wouldn't it be better to use a URL param for the `gameId` instead of sending a body with only that?

Reply:
Yes, I'll change that